src/github.com/getlantern/peerscanner/pk.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
We probably should not be adding a private key to a public repo, third parties could use the compromised key to forge certificates in a MITM situation. I think It would be safer to use the (fixed) certificate we're creating in `peerscanner/web.go`'s `startHttp()` and a public key (of another keypair) instead of sharing the private key and generating a certificate on the fly.